src/python/pants/backend/python/rules/coverage/BUILD

```diff
@@ -0,0 +1,7 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Thoughts on moving `coverage.py` into this new folder too? Then you could separate out the subystem from the rules, for example, with files like `coverage/subsystem.py`, `coverage/rules.py`, etc.

Reply:
I'd really like to put python_test_runner AND coverage in a new dir (rules/ is getting awfully full) but had some issues with mypy. Moving coverage on its own should work fine though.